dev-libs/libmcrypt/libmcrypt-2.5.8.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Replacements for the old Unix crypt"
+DESCRIPTION="libmcrypt are intended to be replacements for the \
+old Unix crypt, except that they are under the GPL and \
+support an ever-wider range of algorithms and modes."
+HOMEPAGE="http://mcrypt.sourceforge.net/"
+COPYRIGHT="1998-2002 Nikos Mavroyanopoulos"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/mcrypt/libmcrypt-$portVersion.tar.gz"
+CHECKSUM_SHA256="e4eb6c074bbab168ac47b947c195ff8cef9d51a211cdd18ca9c9ef34d27a373e"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.4.8"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libmcrypt$secondaryArchSuffix = $portVersion
+	lib:libmcrypt$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libmcrypt${secondaryArchSuffix}_devel = $portVersion
+	devel:libmcrypt$secondaryArchSuffix = $libVersion
+	"
+REQUIRES_devel="
+	libmcrypt$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
```

Comment:
Maybe add a "defineDebugInfoPackage" here?

Reply:
Why not? That's a good idea. Thanks @Begasus !

@nysnatuss, this can be done by inserting these lines before BUILD (with a blank line before and another one after):
```
defineDebugInfoPackage libmcrypt$secondaryArchSuffix \
	$libDir/libmcrypt.so.$libVersion
```